source/pic2card/ui/README.md

```diff
@@ -1,29 +0,0 @@
-# ann-app
-
```

Comment:
keep this file and mark it as deprecated.
update /readme.md
update bug template, workflow

![image](https://user-images.githubusercontent.com/6290692/143953022-ff58963e-03e2-4a3f-b142-4f80ab88b53c.png)
![image](https://user-images.githubusercontent.com/6290692/143953211-e29427f3-4ba3-4809-b2f4-3f21671d2d3e.png)

Reply:
I've updated `/README.md`, but I don't want to leave breadcrumbs around the repo of projects that are no longer hosted here (see, e.g. the old NodeJS visualizer codebase).